FileHandler.gd

```diff
@@ -16,3 +16,61 @@ static func create_directory_if_missing(path: String):
 		var success = dir.make_dir(path)
 		if success != OK:
 			print("Error: Could not create data folder:", path)
+
+static func generate_unique_file_name(base_name: String, extension: String, destination_folder: String) -> String:
+	var unique_name: String = base_name
+	var counter: int = 1
+	var new_file_path: String = destination_folder.plus_file(unique_name + "." + extension)
+
+	while File.new().file_exists(new_file_path):
+		unique_name = base_name + "_" + str(counter)
+		new_file_path = destination_folder.plus_file(unique_name + "." + extension)
+		counter += 1
+	return unique_name + "." + extension
+
+static func copy_supported_image_file(file_path: String, destintation_dir: String) -> String:
+	# These are all supported file types in Godot but may not be supported
+	# by other marker programs.
+	var supported_image_extensions: Array = ["png", "jpg", "jpeg", "bmp", "tga"]
```

Comment:
We should check what other overlay tools support. 
My quick scan of known marker packs has these stats:
.png - ~12k files
.jpeg - 0 files
.jpg - 1 file named "jpgguide.png"
.bmp - 1
.tga - several "mythwrigh**tga**mbit...png"